known-issues.rst

```diff
@@ -74,7 +73,10 @@ RHEL 6.*
 macOS specific
 --------------
 
-macOS 10.12 (Sierra) and OS X 10.11 (El Capitan)
+- Macs must use the :command:`clang` compiler, not :command:`gcc`.
+  :jirab:`DM-3405`
+
+macOS 10.13 (Sierra) and OS X 10.11 (El Capitan)
```

Comment:
Is that 10.13 (High Sierra), 10.12 (Sierra) and 10.11 (El Capitan)? Or two of the three?